gandalf.js

```diff
@@ -7,21 +7,19 @@
  * See README.md for details.
```

Comment:
SCRIPT-Nice job on the comprehensive test coverage, it gives a lot of confidence in the stability of the new feature.